templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+<ul class="sidebar-nav">
+  <li>
+    <a href="{% url 'taxi:index' %}">Home page</a>
+  </li>
+  <li>
+    <a href="#">Manufacturers</a>
+  </li>
+  <li>
+    <a href="#">Cars</a>
+  </li>
+  <li>
+    <a href="#">Drivers</a>
```

Comment:
The links for 'Manufacturers', 'Cars', and 'Drivers' are currently set to `#`, which is a placeholder. Consider updating these links to point to the actual URLs if they are available in your application.

Reply:
They are not available